npm/create-cypress-tests/package.json

```diff
@@ -25,14 +25,14 @@
     "find-up": "5.0.0",
     "fs-extra": "^9.1.0",
     "glob": "^7.1.6",
-    "inquirer": "7.3.3",
+    "inquirer": "8.2.4",
     "ora": "^5.1.0",
```

Comment:
why was this needed?

Reply:
This was necessary as the Lerna upgrade has a dependency upon `inquirer`. 

See this [release](https://github.com/lerna/lerna/releases/tag/v5.2.0)

It was throwing errors when building by leaving it at `7.3.3`

Reply:
What were the inquire breaking changes? Where the scripts in our repo using this dependency tested/check to ensure no other changes were needed?

Reply:
[Version 8 dropped support for Node 10](https://github.com/SBoudrias/Inquirer.js/releases/tag/inquirer%408.0.0)